rest/route/degraded_mode.go

```diff
@@ -41,19 +43,29 @@ func (h *degradedModeHandler) Parse(ctx context.Context, r *http.Request) error
 }
 
 func (h *degradedModeHandler) Run(ctx context.Context) gimlet.Responder {
-	flags, err := evergreen.GetServiceFlags(ctx)
+	config, err := evergreen.GetConfig(ctx)
 	if err != nil {
-		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "retrieving service flags"))
+		return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "retrieving config"))
 	}
-	if flags.CPUDegradedModeDisabled {
-		flags.CPUDegradedModeDisabled = false
-		if err = flags.Set(ctx); err != nil {
+	if config.ServiceFlags.CPUDegradedModeDisabled {
+		config.ServiceFlags.CPUDegradedModeDisabled = false
+		if err = config.ServiceFlags.Set(ctx); err != nil {
 			return gimlet.MakeJSONInternalErrorResponder(errors.Wrap(err, "setting service flags"))
 		}
 		grip.Info(message.Fields{
 			"message": "CPU degraded mode has been triggered",
 			"route":   "/degraded_mode",
 		})
+		if config.Banner == "" {
+			msg := fmt.Sprintf("Evergreen is under high load, max config YAML size has been reduced from %dMB to %dMB. "+
+				"Tasks with >16MB config YAMLs may experience slower scheduling.", config.TaskLimits.MaxParserProjectSize, config.TaskLimits.MaxDegradedModeParserProjectSize)
```

Comment:
nit: `Existing task with > 16 MB...` since I imagine people will be confused how tasks with large YAMLs can run if the YAML size is capped.